gradle/libs.versions.toml

```diff
@@ -75,7 +75,8 @@ kotlin-android = { id = "org.jetbrains.kotlin.android", version.ref = "kotlin" }
 kotlin-multiplatform = { id = "org.jetbrains.kotlin.multiplatform", version.ref = "kotlin" }
 kotlin-serialization = { id = "org.jetbrains.kotlin.plugin.serialization", version.ref = "kotlin" }
 ksp = { id = "com.google.devtools.ksp", version.ref = "ksp" }
-sqldelight = { id = "com.squareup.sqldelight", version.ref = "sqldelight" }
+sqldelight = { id = "app.cash.sqldelight", version.ref = "sqlDelight" }
+
 
```

Comment:
```suggestion

```

Reply:
[toml file, classpath and format](https://github.com/Mindera/Android-KMP-Template/pull/9/commits/4f9fcd593cceb2e74ff2babc507f1eb8b605f576)